docs/src/content/docs/resources/plugins.mdx

```diff
@@ -93,6 +93,11 @@ Extend your site with official plugins supported by the Starlight team and commu
 		title="starlight-site-graph"
 		description="Add an interactive site graph inside your page's sidebar."
 	/>
+	<LinkCard
+		href="https://github.com/trueberryless-org/starlight-sidebar-topics-dropdown"
+		title="starlight-sidebar-topics-dropdown"
+		description="Split your docs page into multiple subpages and switch between them with a dropdown menu in the sidebar."
+	/>
```

Comment:
What do you think about also adding the original plugin as part of this PR? Just realised it hasn’t been added here yet.

```suggestion
	<LinkCard
		href="https://github.com/HiDeoo/starlight-sidebar-topics"
		title="starlight-sidebar-topics"
		description="Starlight plugin to split your documentation into different sections, each with its own sidebar."
	/>
	<LinkCard
		href="https://github.com/trueberryless-org/starlight-sidebar-topics-dropdown"
		title="starlight-sidebar-topics-dropdown"
		description="Split your docs page into multiple subpages and switch between them with a dropdown menu in the sidebar."
	/>
```

Reply:
I love this suggestion! I wanted to add it originally, but didn't want to just take the credit and let HiDeoo do it. Actually I contacted him privately and asked him about it. I'll give him the option that we can include it here right away, let's wait for his answer though...

Reply:
Amazing idea, I totally forgot to do that, thanks you both 🙌